src/util/i18nFormatting.js

```diff
@@ -0,0 +1,30 @@
+function getVariationOfAOrAn(value) {
+  const vowelishLetters = ['a', 'e', 'i', 'o', 'u', 'h'];
+  const isVowelish = vowelishLetters.find(l => value.substring(0, 1) === l);
+  return isVowelish ? 'an' : 'a';
+}
+
+/*
+Arguments:
+value:   The value of the variable that is being passed
+  in to be interpolated
+format:  the pipe-delimited list of formatting flags,
+  it will run the formatting functions sequentially
+  based on the sequence of the flags passed in.
+lng: the language used for the formatting
+NOTE: lng is available(from i18Next in this function) but is not
+currently used, so I've removed it from the arguments list.
+*/
+export default function(value, format) {
+  let newVal = value;
+  const formatFlags = format.split('|');
+  formatFlags.forEach((flag) => {
+    // replace the value of the input value with our new value("a" or "an")
+    if (flag === 'en-handle-an') {
```

Comment:
Maybe overkill for just two flags, but would it be worth having an object that maps the flag name to a transform function? (Kind of like the [error maps](https://github.com/popcodeorg/popcode/blob/master/src/validations/javascript/jshint.js#L32) in the validation module)